config/environments/production.rb

```diff
@@ -1,9 +1,6 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  # overriding the default url option
-  Rails.application.routes.default_url_options[:host] = 'localhost:3001'
```

Comment:
this is the meat of the PR — what I'm hoping will allow the root URL in the deployed app to be that actual app URL